livre/eq/render/RayCastRenderer.cpp

```diff
@@ -430,9 +429,13 @@ struct RayCastRenderer::Impl
 
     void onFrameRender( const NodeIds& bricks )
     {
+        createAndFillVertexBuffer( bricks );
         size_t index = 0;
         for( const NodeId& brick: bricks )
             renderBrick( brick, index++ );
+
+        glDeleteBuffers( 1, &_posVBO );
+        glFlush();
```

Comment:
@eile : I don't agree we should add a function with only one line for deleting. The createAndFillVertexBuffer() is already clear from the name. What can be done is, it can be a const function returning the vbo id and that can be used to delete.